src/add/operator/first.ts

```diff
@@ -2,6 +2,6 @@
 import {Observable} from '../../Observable';
 import {first} from '../../operator/first';
 
-Observable.prototype.first = first;
+Observable.prototype.first = <any>first;
```

Comment:
Tracing it back, it looks like it's because you have an overload in `FirstSignature` with a `void` parameter. I don't think its possible to pass a void value in TypeScript, so that overload can't be called.